workflows/rnaseq.nf

```diff
@@ -137,7 +137,7 @@ salmon_quant_options.args += params.save_unaligned ? " --writeUnmappedNames" : '
 
 def samtools_sort_genome_options  = modules['samtools_sort_genome']
 def samtools_index_genome_options = modules['samtools_index_genome']
-samtools_index_genome_options.args += params.bam_csi_index ? "-c" : ""
+samtools_index_genome_options.args += params.bam_csi_index ? " -c" : ""
```

Comment:
This might be problematic, not sure if there should be a way to avoid possible concatenations without spaces...

Reply:
Yeah....good spot. I know it's a bit dirty but no-one has told me otherwise until now for me to do anything about it 😅 

How's this? https://github.com/nf-core/rnaseq/pull/600/commits/94809839f3842d7d2ccb6ec917047b964c2e66aa